### PR TITLE
feat(explorer): add txs infinite scroll list

### LINKED ISSUE #397: List of transactions
## Story

As a user 
I want to see a list of transactions ordered from most recent to least 
So that I can easily prove my transaction has occurred 

## Acceptance Criteria

- [ ] I can see a continuous list of transactions in order of most recent to least
- [ ] There is some pagination / staggered loading "load more" for speed 
- [ ] For each transaction listed I can see transaction type, submitter, transaction ID and time 

## Tasks

- [ ] Explore and sketch
- [ ] Team and stakeholder review
- [ ] Visual Design
- [x] API support
- [x] Implement as per the infinite scroll component used on the blocks page
- [ ] Team review
- [ ] Etc.

## Sketch

Image

Miro - https://miro.com/app/board/uXjVPVRiSOM=/?moveToWidget=3458764534147549536&cot=14

## Additional details / background info

### PR DESCRIPTION
**Pull request by elmar-vega**
# Related issues 🔗

Closes #397 

# Description ℹ️

Added infinite scrolling list of transactions

# Demo 📺

Screenshot 2022-10-05 at 12 19 32

# Technical 👨‍🔧

- Added block explorer response type and url to env
- Changed render in routes config
- Refactored txs-infinite-list-item and tests

### CODE REVIEW

**Comment by campbellssource** [comment]
request (maybe-blocking): can we have something to click on to open that transaction

File: apps/explorer/.env [hunk 1]
```diff
@@ -6,6 +6,7 @@ NX_VEGA_NETWORKS='{"TESTNET":"https://explorer.fairground.wtf","MAINNET":"https:
 NX_VEGA_ENV=STAGNET3
 NX_GITHUB_FEEDBACK_URL=https://github.com/vegaprotocol/feedback/discussions
 NX_HOSTED_WALLET_URL=https://wallet.testnet.vega.xyz
+NX_BLOCK_EXPLORER=
```
Comment (mattrussell36) [inline apps/explorer/.env:9]: Empty value here?

**Review by mattrussell36 — COMMENTED** [review]
All good, just a question there about the empty env var values for NX_BLOCK_EXPLORER

File: apps/explorer/.env (hunk 1 above)
Comment (elmar-vega) [reply apps/explorer/.env:9]: Put it there, because we don't have values yet, but so we don't forget and not suddenly be surprised why it doesn't work.

**Review by asiaznik — APPROVED** [review]

**Pull request merged**